praw/models/reddit/collections.py

```diff
@@ -441,14 +441,18 @@ def __init__(
         super().__init__(reddit, _data)
         self.subreddit_fullname = sub_fullname
 
-    def create(self, title: str, description: str):
+    def create(self, title: str, description: str, display_layout: str):
```

Comment:
```suggestion
    def create(self, title: str, description: str, display_layout: Optional[str]=None):
```
This should be optional so that it isn't a breaking change.

Reply:
Ah, good point. Totally missed that. I'll make another test case for that empty argument.